drivers/wifi/infineon/Kconfig.cyw43xxx

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) 2022 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig WIFI_CYW43XXX
+	bool "Infineon CYW43xxx SoC Wi-Fi support"
+	select THREAD_CUSTOM_DATA
+	select USE_INFINEON_SDIO
+	select WIFI_OFFLOAD
+	select NET_L2_WIFI_MGMT
+	select WIFI_USE_NATIVE_NETWORKING
+	select ABSTRACTION_RTOS_COMPONENT_ZEPHYR
+	depends on DT_HAS_INFINEON_CYW43XXX_WIFI_SDIO_ENABLED
+	help
+	  Enable Infineon CYW43xxx SoC Wi-Fi support.
+	  Infineon's AIROC™ Wi-Fi & combos portfolio integrates IEEE 802.11a/b/g/n/ac/ax Wi-Fi and Bluetooth® 5.2
+	  in a single-chip solution to enable small-form-factor IoT designs. Combo, standalone Wi-Fi, and Wi-Fi SoCs
+	  with embedded MCU and on-chip networking capabilities are also offered in 1x1 SISO and 2x2 MIMO configurations.
+
+if WIFI_CYW43XXX
+
+# Enable Zephyr variant of Abstraction RTOS
+config ABSTRACTION_RTOS_COMPONENT_ZEPHYR
```

Comment:
The naming of this is super vague, of course it is zephyr, kconfig is running

Reply:
renamed ABSTRACTION_RTOS_COMPONENT_ZEPHYR to INFINEON_ABSTRACTION_RTOS

Reply:
Not a blocking comment anymore, but is there a reason you can't use the `__ZEPHYR__` define? It's not like `CONFIG_ INFINEON_ABSTRACTION_RTOS` is going to be very portable either.

Reply:
Sorry, I didn't realise this was an old symbol you were adding a new default to.
Feel free to revert it back to what it was originally, its a separate issue unrelated to this PR.
If you want to update that in another PR, a name like `HAS_INFINEON_HAL` would match the usage in other HALs